charmcraft.yaml

```
@@ -139,28 +141,38 @@ config:
   options:
     retention-period:
       description: |
-        Maximum trace retention period, in hours. This will be used to configure the compactor to clean up trace data after this time. 
+        Maximum trace retention period, in hours. This will be used to configure the compactor to clean up trace data after this time.
         Defaults to 720 hours, which is equivalent to 30 days. Per-stream retention limits are currently not supported.
       type: int
       default: 720
     always_enable_zipkin:
-      description: Force-enable the receiver for the 'zipkin' protocol in Tempo, even if there is no integration currently requesting it.
+      description:
```

```suggestion
      description: |
```

not sure if needed but we have it like that in the others